website/app/templates/head.hbs

```diff
@@ -16,50 +16,48 @@ We've made some small customizations to simplify this compared to the default te
 
 {{#if this.model.url}}
   <link rel="canonical" href={{this.model.url}} />
-  <meta property="og:url" content={{this.model.url}} />
 {{/if}}
 
 {{! END GENERAL }}
 
-{{! BEGIN OPENGRAPH }}
+{{! BEGIN TWITTER }}
+
+<meta name="twitter:card" content="summary_large_image" />
 
-{{#if this.model.siteName}}
-  <meta property="og:site_name" content={{this.model.siteName}} />
+{{#if this.model.twitterUsername}}
+  <meta name="twitter:site" content={{this.model.twitterUsername}} />
 {{/if}}
 
 {{#if this.model.title}}
-  <meta property="og:title" content={{this.model.title}} />
+  <meta name="twitter:title" content={{this.model.title}} />
 {{/if}}
 
 {{#if this.model.description}}
-  <meta property="og:description" content={{this.model.description}} />
+  <meta name="twitter:description" content={{this.model.description}} />
 {{/if}}
 
 {{#if this.model.imgSrc}}
-  <meta property="og:image" content={{this.model.imgSrc}} />
-  <meta property="og:image:type" content="image/jpg" />
+  <meta name="twitter:image" content={{this.model.imgSrc}} />
```

Comment:
```suggestion
  <meta name="twitter:image" content={{concat (this.model.url + '/' + this.model.imgSrc}} />
```
something like this

Reply:
Thanks. The imgSrc includes "/" so I did: 
`<meta name="twitter:image" content={{concat this.model.url this.model.imgSrc}} />`

In environment.js:
```
imgSrc: '/assets/logos/share-card2.jpg',
url: 'https://hds-website-git-hds-1383-twitter-preview-hashicorp.vercel.app',
```